SimPEG/FLOW/Richards/Empirical.py

```diff
@@ -1,13 +1,15 @@
 from SimPEG import Mesh, Maps, Utils, np
 
+from six import add_metaclass
 
+@add_metaclass(Utils.SimPEGMetaClass)
 class NonLinearMap(object):
     """
     SimPEG NonLinearMap
 
     """
 
-    __metaclass__ = Utils.SimPEGMetaClass
+    #__metaclass__ = Utils.SimPEGMetaClass
```

Comment:
We can probably kill these lines. In fact the whole `SimPEGMetaClass` actually needs to be removed.

![image](https://cloud.githubusercontent.com/assets/913249/17860743/b860debc-684b-11e6-8dd1-538c237a8fd8.png)

Rather silly.